assets/i18n/en.json

```diff
@@ -866,7 +866,7 @@
     "I18N_SIDEBAR_VOLUNTEER_DESCRIPTION": "Join our global team to create and improve lessons.",
     "I18N_SIGNIN_LOADING": "Signing in",
     "I18N_SIGNIN_PAGE_TITLE": "Sign in",
-    "I18N_SIGNUP_AGREE_LICENSE_DESCRIPTION": "By checking the box to the left of this text, you acknowledge, agree, and accept to be bound by the <[sitename]> Terms of Use, found <a href=\"/terms\" target=\"_blank\">here</a>.",
+    "I18N_SIGNUP_AGREE_LICENSE_DESCRIPTION": "&nbsp;By checking the box to the left of this text, you acknowledge, agree, and accept to be bound by the <[sitename]> Terms of Use, found <a href=\"/terms\" target=\"_blank\">here</a>.",
```

Comment:
Instead of modifying the translation, can we fix the corresponding HTML file so there is sufficient padding next to the checkbox?